contracts/token/LinearVestingWithPenalty.sol

```diff
@@ -415,37 +280,122 @@ contract LinearVestingWithPenalty is Initializable, ERC20Upgradeable, AccessCont
         return vestingSchedules[beneficiary].length;
     }
 
+
     /**
-     * @dev Transfers a beneficiary's vesting schedules and token balance to another address.
-     * @param from The address of the current beneficiary.
-     * @param to The address of the new beneficiary.
-     */
-    function transferBeneficiary(address from, address to) external onlyRole(ADMIN_ROLE) {
-        if (from == address(0) || to == address(0)) revert InvalidRecipientAddress();
-        if (from == to) revert CannotTransferToSelf();
+    * @dev Claims the full vested tokens for the caller from specific vesting schedules and burns the claimed vSOPH tokens.
```

Comment:
This function claims the _releasable_ amount (`= full vested - released`). Adding this to clarify the distinction with `claimSpecificSchedulesWithPenalty`

Reply:
fixed